manifests/ha/install.yaml

```diff
@@ -2013,14 +2049,14 @@ spec:
 apiVersion: v1
 kind: Service
 metadata:
-  annotations:
-    service.alpha.kubernetes.io/tolerate-unready-endpoints: "true"
+  annotations: null
```

Comment:
I know you didn't do this, but this looks like a pretty bad kustomize bug. Nulling out annotations will cause defaulted annotations from controllers to get wiped out. Another reason why we may not be able to migrate to kustomize v2.1 yet